pkg/controller/apis/config/types.go

```diff
@@ -18,7 +18,8 @@ package config
 
 import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	componentbaseconfig "k8s.io/component-base/config"
+	config2 "k8s.io/controller-manager/config"
```

Comment:
Can we make this cmconfig rather than config2? We may also want to alias service/config as serviceconfig. If we did that then controller-managers config could just be config. As the base config that would seem to make more sense.

Reply:
Updated to use `cmconfig` for /control-manager/config and `serviceconfig` for /service/config